lib/lorekeeper/multi_logger.rb

```diff
@@ -26,7 +37,7 @@ def respond_to?(method)
     end
 end
 
-    def method_missing(method, *args, &block)
```

Comment:
After I renamed the  method_missing method two specs started failing
[One of them](https://github.com/JordiPolo/lorekeeper/blob/d3bcff1ae3c6600cafc79bec34cd801d4ea1b3a9/spec/lib/lorekeeper/multi_logger_spec.rb#L24) checks whether `#write`  method can be called from Multilogger.

But `#write` belongs to a private class within `FastLogger` and the method is called within the `#add` method.

I believe the `#add` method is the main one to implement and it’s the entry point for any logger class, whereas `#write` and other methods are not meant to be called by from outside.

So, I kept the `method_missing` renamed (as suggested) and I removed [the spec that checks](https://github.com/JordiPolo/lorekeeper/pull/26/files#diff-3ec923edcde18eaf057a5e7a8de440fd02df05fac0b2e63022437bc8a83d684fL24) whether we can call `#write` method.

Reply:
I don't think we should drop the `write` method support. Some loggers need it.